src/main/java/io/cdap/e2e/pages/actions/CdfSysAdminActions.java

```diff
@@ -77,4 +84,237 @@ public static void verifySuccess() {
     }
     Assert.assertTrue(checkParam);
   }
+
+  /**
+   * Click on the System Admin Menu
+   */
+  public static void clickOnSystemAdminMenu(String menuName) {
+    ElementHelper.clickOnElement(CdfSysAdminLocators.locateMenu(menuName));
+  }
```

Comment:
Few things here,

1. If we are simply using this method to select the System menu every type we don't have to pass the parameter.
2. We shouldn't actually make this method or Step name specfic to System Admin make it re-usable to some extent it is right now.

Changes I would suggest.
1. Change the method name it shouldn't be page specific if method is common.
2.Update the step, it should accept one parameter just the definition will be a bit changed.

ex : ``` Open "System Admin" menu``` or ``` Open "Hub" page *change*```

Reply:
Changed the step definition, parameter was already added.